catpol/spiders/cdep_career_history.py

```diff
@@ -0,0 +1,44 @@
+import scrapy
+import logging
+
+import sys
+from scrapy.http import Response
+
+import catpol.loaders as loaders
+import catpol.items as items
+import catpol.http as http
+import string
+import catpol.cmdinput as cmdinput
+from typing import Sequence, TypeVar, Coroutine
+
+
+class CdepCareerHistory(scrapy.Spider):
```

Comment:
I was asked by Razvan to create a new spider in order to map the movements of the politicians from one political party to the other in order to spot some behaviors, etc.
I think it's a good programming practice to stick to the [Single Responsability Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle).